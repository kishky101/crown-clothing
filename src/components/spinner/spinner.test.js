import renderer from 'react-test-renderer';
import Spinner from './spinner.component';

test('creating spinner snapshot', () => {
    const spinner = renderer.create(<Spinner />).toJSON;
    expect(spinner).toMatchSnapshot();
})