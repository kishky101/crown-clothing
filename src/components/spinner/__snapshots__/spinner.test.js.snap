// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`creating spinner snapshot 1`] = `[Function]`;
